examples/scripts/create-model.py

```diff
@@ -0,0 +1,125 @@
+# This script is intended to be a stripped back version of the
+# 'examples/notebooks/create-model.ipnb' so for more details please see
+# that notebook
+
+import pybamm
+import numpy as np
+import autograd.numpy as auto_np
+import matplotlib.pyplot as plt
+
+# 1. Initialise model ------------------------------------------------------------------
+model = pybamm.BaseModel()
+
+# 2. Define parameters and variables ---------------------------------------------------
+# dimensional parameters
+k_dim = pybamm.Parameter("Reaction rate constant")
+L_0_dim = pybamm.Parameter("Initial thickness")
+V_hat_dim = pybamm.Parameter("Partial molar volume")
+c_inf_dim = pybamm.Parameter("Bulk electrolyte solvent concentration")
+
+
+def D_dim(cc):
+    return pybamm.FunctionParameter("Diffusivity", cc)
+
+
+# dimensionless parameters
+k = k_dim * L_0_dim / D_dim(c_inf_dim)
+V_hat = V_hat_dim * c_inf_dim
+
+
+def D(cc):
+    c_dim = c_inf_dim * cc
+    return D_dim(c_dim) / D_dim(c_inf_dim)
+
+
+# Define variables
+c = pybamm.Variable("Solvent concentration", domain="negative electrode")
+L = pybamm.Variable("SEI thickness")
+
+# 3. State governing equations ---------------------------------------------------------
+R = -k * pybamm.BoundaryValue(c, "left")  # SEI reaction flux
+N = -(1 / L) * D(c) * pybamm.grad(c)  # solvent flux
+dcdt = -(1 / L) * pybamm.div(N)  # solvent concentration governing equation
+dLdt = -V_hat * R  # SEI thickness governing equation
+
+model.rhs = {c: dcdt, L: dLdt}  # add to model
+
+# 4. State boundary conditions ---------------------------------------------------------
+grad_c_left = L * R / D(pybamm.BoundaryValue(c, "left"))  # left bc
```

Comment:
Note that you need to do `pybamm.BoundaryValue(D(c), "left")` (because `D` is non-linear) to get the right accuracy for the boundary condition. Might be worth mentioning this in the tutorial.